pds_doi_service/core/util/test/config_parser_test.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import sys
+import os
+from pds_doi_service.core.util.config_parser import DOIConfigUtil
+
+class ConfigParserTest(unittest.TestCase):
+    def test_add_absolute_path(self):
+
+        parser = DOIConfigUtil().get_config()
+        self.assertEqual(parser['OTHER']['db_file'], os.path.join(sys.prefix, 'doi.db'))
+        self.assertEqual(parser['OTHER']['transaction_dir'], os.path.join(sys.prefix, 'transaction_history'))
+
```

Comment:
This could also use a test with a config that specifies `db_file` and `transaction_dir` as absolute paths to start with